docs/beta/grain.md

```diff
@@ -1,30 +1,44 @@
 
-Projects can choose to issue their Grain as a cryptocurrency or token, for example by issuing Grain as an ERC-20 token on Ethereum. This allows Grain to be a conduit with which a project can pay or financially reward contributors.
+Projects can choose to issue their Grain as a cryptocurrency or token, for example by issuing Grain as an ERC-20 token on Ethereum. This allows Grain to be a conduit with which a project can pay or financially reward participants.
 
 Note that Grain does not _have_ to be tied to any actual monetary concept, such as cryptocurrency. A SourceCred project can name it “Points” and use it solely as a scoring system in addition to Cred values.
 
 ## How you earn Grain
 
-Grain is created during what we call “Harvests.” A Harvest is a periodic cycle in which the project produces new Grain tokens, and distributes those to contributors based on their Cred scores. Your Cred score determines the percentage of Grain you receive out of the total amount allocated to the community. 
+Grain is created during what we call “Harvests.” A Harvest is a two-fold process with periodic cycles, which can be configured independently:
 
-Grain Harvests can be allocated to a community as often or as seldom as that community wishes. Furthermore, each Grain distribution (Harvest) can vary in amount. How the Grain is produced and distributed is up to the project. For example, a project could produce a fixed amount of Grain every week and distribute it to contributors based on their recent contributions, or it could have a target “Grain-per-Cred” payout ratio, and Harvest enough to ensure that all contributors receive at least that amount.
+#### 1. Cred Calculation
+In this cycle, the project recomputes the Cred Contribution Graph by gathering the data your plugins are tracking, and calculating new Cred scores. By default, this cycle runs every 6 hours.
+
+#### 2. Grain Distribution
+In this less-frequent cycle, the project distributes Grain based on participants’ newest Cred scores. By default, this cycle runs once a week.
+
+Your Cred score determines the percentage of Grain you receive out of the total amount allocated to the community for each Harvest, regardless of how frequently Cred Calculation happens within it. 
+
+#### Grain Distribution Policies
+SourceCred currently has two policies for how a project distributes Grain:
+
+- `IMMEDIATE` splits Grain evenly based on everyone’s Cred earned per the most recent Grain Distribution cycle. (default: last week)
+- `BALANCED` splits Grain based on lifetime Cred. This will accommodate a retroactively-updated Cred shift by equalizing compensation as soon as possible, using as few Harvests as possible to achieve lifetime distribution balance. Note: This means that overpaid contributors might receive 0 Grain for some period of time, if it takes more than one full Harvest worth of Grain distribution to achieve balance.
```

Comment:
```suggestion
- `BALANCED` splits Grain based both on lifetime Cred and on lifetime Grain earnings. Balanced tries to ensure that everyone in the project receives a total Grain payment which is consistent with their total Cred score. For example, suppose that a contributor used to have a low Cred score, and as such received a small amount of Grain. However, the community recently changed its weights, or added a new plugin, such that the contributor now has a larger amount of Cred. The balanced policy sees that this contributor is underpaid, so it will pay them extra to "catch them up" to others in the project. Conversely, contributors might be "overpaid", and they'll receive less Grain until the payouts have been equalized.
```